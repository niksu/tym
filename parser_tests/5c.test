% Example of mutual recursion.

c().
a().
b() :- c().
b() :- b().

c() :- b().
