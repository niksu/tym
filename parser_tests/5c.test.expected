input contents |% Example of mutual recursion.

c().
a().
b() :- c().
b() :- b().

c() :- b().
|
(no query given)
stringed file contents (size=126, remaining=2874)
|predicate_c().
predicate_a().
predicate_b() :- predicate_c().
predicate_b() :- predicate_b().
predicate_c() :- predicate_b().|
stringed file contents strlen=125
